tools/bdf-to-mono/src/main.rs

```diff
@@ -2,19 +2,19 @@ use std::path::PathBuf;
 
 use bdf_parser::BdfFont;
 use bdf_to_mono::{Encoding, MonoFontData};
-use clap::Clap;
+use clap::Parser;
 
-#[derive(Clap)]
+#[derive(clap::Parser)]
 struct Args {
-    #[clap(about = "BDF input")]
+    #[clap(help = "BDF input")]
```

Comment:
I haven't tried to run this, but the `Cargo.toml` still specifies `clap` version `3.0.0-beta.2` and I think these changes were implemented in a later `clap` version. I would be OK with just leaving this crate as it was, because someday I might actually finish the replacement tool in the `bdf` repo.

Reply:
Hmm, if I lock the version to `"=3.0.0-beta.2"` (confirmed with `cargo tree`) and revert to the old code, I see the same compile errors as before this PR, so I'm not sure what's going on there, potentially caused by `cargo update`? I can revert this change and dig into why these errors are popping up but if this tool will be replaced eventually anyway, should we spend much time on it?

Reply:
No, don't spend any time on this and commit whatever works.